com.unity.cinemachine/Runtime/Core/TargetPositionCache.cs

```diff
@@ -252,11 +264,15 @@ public static Vector3 GetTargetPosition(Transform target)
         }
 
         /// <summary>
-        /// If Recording, will log the target rotation at the CurrentTime.
-        /// Otherwise, will fetch the cached position at CurrentTime.
+        /// When using Timeline in Edit mode:
+        ///  - If Recording, will log the target rotation at the CurrentTime.
+        ///  - Otherwise, will fetch the cached rotation at CurrentTime.
+        ///  
+        /// In Play mode, and when not scrubbing timeline:
+        ///  - will return the rotation directly from the transform.
```

Comment:
```suggestion
        /// When using Timeline in Edit mode:
        ///  - If you're Recording, the method logs the target position at the CurrentTime.
        ///  - Otherwise, it fetches the cached position at CurrentTime.
        ///  
        /// When using Timeline in Play mode, and when you're not scrubbing it:
        ///  - The method returns the position directly from the Transform.
```